### PR TITLE
links: graph-store rewrite

### PR DESCRIPTION
**Pull request by tacryt-socryp**
Completed:
- Link submission
- Link commenting
- Rendering data coming from graph-store
- Sidebar rendering (required create thread)
- Settings metadata editing
- Finished remove / delete support
- Hooks
- Joining (requires join thread to be done)

Edit: Completely done

### CODE REVIEW

**Review by liam-fitzgerald — CHANGES_REQUESTED** [review]
couple of nits, looks good otherwise

File: pkg/interface/src/views/apps/links/components/new.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+          name,
+          description,
+          group
+        );
+      } else {
+        await props.api.graph.createUnmanagedGraph(
+          resourceId,
+          name,
+          description,
+          { open: {
```
Comment (liam-fitzgerald) [inline pkg/interface/src/views/apps/links/components/new.tsx:46]: This differs from the current behaviour of unmanaged links, which are always invite-only

**Review by liam-fitzgerald — CHANGES_REQUESTED** [review]
Fetch-on-every render still an issue from the old review and one small nit, but otherwise looks good!

File: pkg/interface/src/views/apps/links/components/new.tsx (hunk 1 above)
Comment (tacryt-socryp) [reply pkg/interface/src/views/apps/links/components/new.tsx]: Behavior is now the same, thanks!

**Comment by tacryt-socryp** [comment]
@liam-fitzgerald I've fixed the fetch every render issue with `useEffect` as you suggested. Thanks!

**Review by belisarius222 — APPROVED** [review]
Hoon LGTM.  I don't really have enough context on the JS to review thoroughly.

**Comment by tacryt-socryp** [comment]
Currently, joining from the Groups page doesn't work, as the metadata tag "graph" doesn't correspond to the use case... Going to work on adding the proper tag to graphs in the `%create` thread, and making the redirect route in `/~graph` check for that tag before redirecting to the proper app.

**Comment by matildepark** [comment]
Aren't all the routes changing in the other PR anyway? I don't know what the new join routes are in that one.

**Comment by tacryt-socryp** [comment]
This PR is good to go as is. I need to do some `%metadata-store` work to make the “Channels” page work normally with graph-store resources, and that does block release of the new Links version, but is not important within context of this PR.

**Pull request merged**